### PR TITLE
header added

### PR DESCRIPTION
**Pull request by AlNeon**
- [DEMO LINK](https://AlNeon.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://AlNeon.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,62 @@
+html {
+  --header-height: 60px;
+  --header-margin: 50px;
```
Comment (sTorba24) [inline src/style.css:3]: better to declare variables in :root sleector

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,62 @@
...
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  position: relative;
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
```
Comment (sTorba24) [inline src/style.css:31]: set it for html tag

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Generally it's cool! Some little improvements

**Comment by AlNeon** [comment]
everything fixed

File: src/index.html [hunk 3]
```diff
@@ -6,9 +6,50 @@
...
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" data-qa="hover"
+              href="#laptops&computers">Laptops & Computers</a>
```
Comment (etojeDenys) [inline src/index.html:37]:
```suggestion
            <a 
              class="nav__link" 
              data-qa="hover"
              href="#laptops&computers"
            >
              Laptops & Computers
            </a>
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]
image
"apple" should be blue

File: src/index.html (hunk 3 above)
Comment (AlNeon) [reply src/index.html:37]: fixed one, broke the other..)
all fixed, thank you!

**Comment by AlNeon** [comment]
v.3

File: src/index.html [hunk 4]
```diff
@@ -5,10 +5,56 @@
...
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            class="nav__link"
+            data-qa="hover"
+            href="#laptops&computers">Laptops & Computers</a>
```
Comment (etojeDenys) [inline src/index.html:42]:
```suggestion
            <a
              class="nav__link"
              data-qa="hover"
              href="#laptops&computers"
            >
              Laptops & Computers
            </a>
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]
fix is-active line
image

**Comment by etojeDenys** [comment]
> fix is-active line image

or feel free to ask for some help in chat

**Comment by AlNeon** [comment]
v.4

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**